config/checkstyle_checks.xml

```diff
@@ -399,6 +399,12 @@
       //LITERAL_CATCH//METHOD_CALL[.//IDENT[@text = 'printStackTrace']]/.."/>
       <message key="matchxpath.match" value="Avoid using 'printStackTrace'."/>
     </module>
+    <module name="MatchXpath">
+      <property name="query" value="//CLASS_DEF//OBJBLOCK//METHOD_DEF//MODIFIERS//
+            ANNOTATION[.//IDENT[@text='Test']]//ANNOTATION_MEMBER_VALUE_PAIR
```

Comment:
why are there so many `//`s, which basically amount to a wildcard depth search?
Example: `METHOD_DEF//MODIFIERS` seems excessive seeing as they are right next to each other in https://github.com/checkstyle/checkstyle/pull/9145#issuecomment-756313145 .

I also question if we need the start `CLASS_DEF//OBJBLOCK`.

Isn't:
````
      <property name="query" value="//METHOD_DEF/MODIFIERS//
            ANNOTATION[.//IDENT[@text='Test']]/ANNOTATION_MEMBER_VALUE_PAIR
            [./IDENT[@text='expected']]"/>
````

more than enough?

Reply:
I have confirmed that this query works, fixed.